DigitalOffice/Services/ProjectService/Database/Entities/ProjectManagerUser.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace ProjectService.Database.Entities
+{
+    public class ProjectManagerUser
```

Comment:
Idk if "user" suffix is necessary. Or is it?

Reply:
Yes it is. We discussed this on our previous call: we mark our properties so that we understand where does this data service come from.